Lombiq.Hosting.Tenants.FeaturesGuard/Handlers/FeaturesEventHandler.cs

```diff
@@ -123,7 +123,7 @@ private Task HandleConditionallyEnabledFeaturesAsync()
         out HashSet<string> featuresToEnable,
         out HashSet<string> featuresToDisable)
     {
-        if (!conditionallyEnabledFeatures.Any())
+        if (conditionallyEnabledFeatures?.Any() != true)
```

Comment:
Instead of this null check please make a default instance of the Options' property in the Model itself.
![image](https://github.com/Lombiq/Hosting-Tenants/assets/92299130/50f106dd-48fc-4b5e-a85b-39221678ccca)